packages/compass-import-export/src/utils/formatters.js

```diff
@@ -7,23 +7,24 @@ import { EJSON } from 'bson';
 import { serialize as flatten } from './bson-csv';
 import { Transform } from 'stream';
 import { EOL } from 'os';
-
+import pick from 'lodash.pick';
 /**
  * @returns {Stream.Transform}
  */
-export const createJSONFormatter = function({ brackets = true } = {}) {
+export const createJSONFormatter = function({ brackets = true, columns = [] } = {}) {
```

Comment:
I think all the filtered fields are selected with a projection, shouldn't get here